lib/utils/SuitestError.js

```diff
@@ -34,7 +32,7 @@ class SuitestError extends Error {
 		if (response) {
 			this.response = response;
 		}
-		this.stack = logLevel === logLevels.debug ? this.stack : stripSuitestCodeFromStack(this.stack);
```

Comment:
this logic moved to another place?

Reply:
no, that's hard to do, at least I don't know the easy way, for end user it might be useless, we can import default loglevel from config file but it will be used on all instances?